base.py

```diff
@@ -190,9 +191,22 @@ def __init__(self, addr, username, password, create_test_user=True,
     def __del__(self):
         self.browser.quit()
 
+    def _cleanup_string(self, string):
+        tags = re.compile('<.*?>')
+        return re.sub(tags, '', string).strip()
```

Comment:
I guess `self` instance shouldn't be passed to the method until it doesn't use it.